src/registrar/models/draft_domain.py

```diff
@@ -20,3 +20,5 @@ def __str__(self) -> str:
         default=None,  # prevent saving without a value
         help_text="Fully qualified domain name",
     )
+
+    is_complete = models.BooleanField(default=True, help_text="Determines if this Draft is complete or not")
```

Comment:
I think we might me able to do without is_complete. Trick would be to refactor what we're passing to the modal and to give it a 'New domain request {{ application.created_at }}' - which may be tricky, but worth it.